dev.stewlab.PortaWar.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Version=1.6.0
```

Comment:
this vesion field is invalid. This is NOT the application version.

Reply:
resolved